src/css/globals.css

```diff
@@ -1,8 +1,38 @@
 @tailwind base;
 
+@layer base {
+  :root {
+    --background-800: #f3f4f6;
+    --background-700: #ffffff;
+    --foreground: #4b5563e6;
+
+    --primary: #0e7490;
+
+    --success: #16A34A;
+    --warning: #f97316;
+    --danger: #ef4444;
+    --info: #06b6d4;
+    --misc: #7c3aed;
+  }
+
+  .dark {
+    --background-800: #323b44;
+    --background-700: #3a444e;
+    --foreground: #94a3b8;
+
+    --primary: #0e7490b3;
+
+    --success: #16A34A;
+    --warning: #f97316b3;
+    --danger: #ef4444b3;
+    --info: #06b6d4b3;
+    --misc: #7c3aed;
+  }
```

Comment:
to add a new theme, just add a new `.<name>` class and let the user select the theme with the ThemeButton. This makes it better to improve branding later with a more elaborated palette of colors